vulnerabilities/importers/openssl.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) 2017 nexB Inc. and others. All rights reserved.
```

Comment:
:nit Remove the `2017`, I know we have that same outdated license header everywhere, which need to be updated as well.

Reply:
Should I change it to 2020 or just remove the year?